dailybeast/mobiledoc-react-renderer
# [FEATURE] Updated dependencies

Set react and react-dom as devDependencies and react as a peerDependency to avoid multiple copies of react in projects consuming this package

# How to test
1. Clone repo
2. `cd` into the repo and make sure you run `nvm use`
3. Run `npm install`
4. Run `npm pack` which will create a `.tgz` file with the name `dailybeast-mobiledoc-react-renderer-1.1.1.tgz`
5. Open up https://github.com/dailybeast/thedailybeast in your command line
6. From within that repo, `npm install relative/path/to/this/repo/dailybeast-mobiledoc-react-renderer-1.1.1.tgz`
7. Run `npm run dev` from the daily beast repo and ensure that everything works as expected
